cinema/views.py

```diff
@@ -34,6 +47,36 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        """Converts a list of string IDs to a list of integers"""
+        return [int(str_id) for str_id in qs.split(",")]
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if genres:
+            genres_ids = self._params_to_ints(genres)
+            queryset = queryset.filter(genres__id__in=genres_ids)
+            if self.action in ("list", "retrieve"):
+                queryset = queryset.prefetch_related("genres")
+        if actors:
+            actors_ids = self._params_to_ints(actors)
+            queryset = queryset.filter(actors__id__in=actors_ids)
+            if self.action in ("list", "retrieve"):
+                queryset = queryset.prefetch_related("actors")
```

Comment:
it's unnecessary also

Reply:
"There should be implemented def validate function in the serializer. UniqueTogether you can add to the model's Meta."
Why should I  implement def validate if this UniqueTogetherValidator does a great job? I checked everything is working

Reply:
<img width="1440" alt="Знімок екрана 2022-11-27 о 17 51 37" src="https://user-images.githubusercontent.com/106813923/204151682-ae8373b5-3d49-4d9b-9ae4-426624fee2af.png">

Reply:
Because I saw that you implemented  `validate_seat` method in models and didn't use it anywhere, so I decided that you want to use it in serializer )

Reply:
![image](https://user-images.githubusercontent.com/104218550/204242814-5309d926-255e-4459-b874-c043b606594f.png)